yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/factory/UnsignedPathFactory.java

```diff
@@ -20,6 +20,7 @@ public class UnsignedPathFactory {
     private static final String DOCS_UPLOAD_FACE_CAPTURE_IMAGE_PATH_TEMPLATE = "/sessions/%s/resources/face-capture/%s/image?sdkId=%s";
     private static final String DOCS_TRIGGER_IBV_NOTIFICATION_PATH_TEMPLATE = "/sessions/%s/instructions/email?sdkId=%s";
     private static final String IDENTITY_SESSION_CREATION_TEMPLATE = "/v2/sessions";
```

Comment:
I just notice here but  `"/v2/sessions"` is not a template
And then I am thinking, do we really need the `_PATH_TEMPLATE` and `_TEMPLATE` prefix in any of these values? 
I'd just remove then all
```
    private static final String PROFILE = "/profile/%s?appId=%s";
    private static final String AML_CHECK = "/aml-check?appId=%s";
    private static final String QR_CODE_APP = "/qrcodes/apps/%s";
    private static final String DOCS_CREATE_SESSION = "/sessions?sdkId=%s";
    ....
```

Reply:
I know it is ugly but I found it this way and I'm just sticking with it. The best approach here would be to create an enum. This is one of the many things that need to be changed at the next major release (breaking changes)

Reply:
Fine, I just mentioned it because it is a small self-contained change, as these are all private variables. 
#[Boy scout rule](https://biratkirat.medium.com/step-8-the-boy-scout-rule-robert-c-martin-uncle-bob-9ac839778385) :)